[2024-12-19T13:16:02.139+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T13:16:02.146+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:16:00.283338+00:00 [queued]>
[2024-12-19T13:16:02.149+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:16:00.283338+00:00 [queued]>
[2024-12-19T13:16:02.149+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T13:16:02.154+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): ExecuteNotebook> on 2024-12-19 13:16:00.283338+00:00
[2024-12-19T13:16:02.158+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=12555) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T13:16:02.159+0000] {standard_task_runner.py:72} INFO - Started process 12572 to run task
[2024-12-19T13:16:02.158+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'PRCS_TEST_FRAMEWORK1', 'ExecuteNotebook', 'manual__2024-12-19T13:16:00.283338+00:00', '--job-id', '2328', '--raw', '--subdir', 'DAGS_FOLDER/script/PRCS_TEST_FRAMEWORK1.py', '--cfg-path', '/tmp/tmpgo4gnfhx']
[2024-12-19T13:16:02.160+0000] {standard_task_runner.py:105} INFO - Job 2328: Subtask ExecuteNotebook
[2024-12-19T13:16:02.178+0000] {task_command.py:467} INFO - Running <TaskInstance: PRCS_TEST_FRAMEWORK1.ExecuteNotebook manual__2024-12-19T13:16:00.283338+00:00 [running]> on host 60778319043b
[2024-12-19T13:16:02.208+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Nattapat-process' AIRFLOW_CTX_DAG_ID='PRCS_TEST_FRAMEWORK1' AIRFLOW_CTX_TASK_ID='ExecuteNotebook' AIRFLOW_CTX_EXECUTION_DATE='2024-12-19T13:16:00.283338+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-19T13:16:00.283338+00:00'
[2024-12-19T13:16:02.208+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T13:16:02.215+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-12-19T13:16:10.492+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-12-19T13:16:10.498+0000] {logging_mixin.py:190} INFO - Success Running : http://localhost:8888/notebooks/Log_output/PRCS_TEST_FRAMEWORK1/test1_executed.ipynb
[2024-12-19T13:16:10.498+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-19T13:16:10.502+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T13:16:10.502+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=PRCS_TEST_FRAMEWORK1, task_id=ExecuteNotebook, run_id=manual__2024-12-19T13:16:00.283338+00:00, execution_date=20241219T131600, start_date=20241219T131602, end_date=20241219T131610
[2024-12-19T13:16:10.508+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T13:16:10.513+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T13:16:10.514+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
